.github/workflows/doc.nuxeo.com.yml

```
@@ -0,0 +1,66 @@
+name: Build and Deploy doc.nuxeo.com
+
+on:
+  push:
+    branches:
+      - master
+      - development
+
+jobs:
+  documentation_ci:
+    name: Build & Publish documentation to ${{ github.ref_name == 'master' && 'production' || github.ref_name }}
+    runs-on: doc
+    environment:
+      name: ${{ github.ref_name == 'master' && 'production' || github.ref_name }}
+    steps:
+      - name: Set up SSH
+        uses: webfactory/ssh-agent@d4b9b8ff72958532804b70bbe600ad43b36d5f2e  #v0.8.0
+        with:
+          ssh-private-key: ${{ secrets.SERVER_SSH_PRIVATE_KEY }}
+
+      - name: Checkout code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
+
+      - name: Set up Node.js
+        uses: actions/setup-node@8f152de45cc393bb48ce5d89d36b731f54556e65 #v4.0.0
+        with:
+          node-version: '14'
```

```suggestion
      - name: Set up SSH
        uses: webfactory/ssh-agent@d4b9b8ff72958532804b70bbe600ad43b36d5f2e # v0.8.0
        with:
          ssh-private-key: ${{ secrets.SERVER_SSH_PRIVATE_KEY }}

      - name: Checkout code
        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1

      - name: Set up Node.js
        uses: actions/setup-node@8f152de45cc393bb48ce5d89d36b731f54556e65 # v4.0.0
        with:
          node-version: '14'
```

not sure if not doing this could break dependabot